### PR TITLE
Fix docker

### PR DESCRIPTION
**Pull request by mbucknell**
Before making a pull request
----------------------------

- [X] Make sure all tests run
- [X] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by mbucknell** [comment]
In addition to fixing the graph-server Dockerfile so that it would work with Docker 19.0.x, I removed the docker-compose files which were not working, the assets/Dockerfile as it is only used by the docker-compose files. Also removed obsolete code from graph-server.

All graph-server dependencies should be up to date.

File: graph-server/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-        chromium@edge \
-        harfbuzz@edge \
-        nss@edge
-
-
-# Tell Puppeteer to skip installing Chrome. We'll be using the installed package.
-ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true \
-    CHROME_BIN=/usr/bin/chromium-browser \
-    CHROME_PATH=/usr/lib/chromium/
+FROM node:10.17.0-slim
```
Comment (ayan-usgs) [inline graph-server/Dockerfile]: Going back an forth about stretch vs buster. Let's leave it as it is.

File: Jenkinsfile.deploy [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@Library(value='PipelineLibs@0.7.0', changelog=false) _
+@Library(value='PipelineLibs@2.0.3', changelog=false) _
```
Comment (ayan-usgs) [inline Jenkinsfile.deploy]: Might be worth switching this over to the IOW fork.

Comment (mbucknell) [reply Jenkinsfile.deploy]: Agreed, I will try that.

**Pull request merged**